lib/node_modules/@stdlib/stats/base/dists/triangular/mgf/benchmark/benchmark.native.js

```diff
@@ -56,10 +56,10 @@ bench( pkg+'::native', opts, function benchmark( b ) {
 	mode = new Float64Array( len );
 
 	for ( i = 0; i < len; i++ ) {
```

Comment:
```suggestion
	mode = new Float64Array( len );
	for ( i = 0; i < len; i++ ) {
```

Same comment as above and applies elsewhere